addons/binding/org.openhab.binding.networkhealth/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="networkhealth"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--NetworkHealth Binding-->
+    <thing-type id="DEVICE">
```

Comment:
As per the "reference implementation" https://github.com/eclipse/smarthome/blob/master/binding/org.eclipse.smarthome.binding.yahooweather/ESH-INF/thing/thing-types.xml, I would suggest to change all thing type ids, channel ids, and config parameter names to lower case - thanks!